lib/qmp.ml

```diff
@@ -143,10 +147,22 @@ let message_of_string str =
   in
 
   let event json =
-    let ts x = json |> U.member "timestamp" |> U.member x |> U.to_int in
-    { timestamp = (ts "seconds", ts "microseconds")
-    ; event     = json |> U.member "event" |> U.to_string
-    }
+    let event' json =
+      let ts x = json |> U.member "timestamp" |> U.member x |> U.to_int in
+      { timestamp = (ts "seconds", ts "microseconds")
+      ; event     = json |> U.member "event" |> U.to_string
+      }
+    in
+    let rtc_offset json =
```

Comment:
rtc_offset() is used only inside event_data(), so I think it's better if it's inside event_data()

Reply:
Thanks, updated